ui/src/app/pages/analysis/runtime-panel.spec.tsx

```diff
@@ -1147,22 +1147,33 @@ describe('RuntimePanel', () => {
         numberOfWorkers: 2,
         numberOfPreemptibleWorkers: 0,
         workerMachineType: 'n1-standard-4',
-        workerDiskSize: DATAPROC_WORKER_MIN_DISK_SIZE_GB,
+        workerDiskSize: DATAPROC_MIN_DISK_SIZE_GB,
       },
     });
 
     const wrapper = await component();
 
-    const costEstimator = () => wrapper.find('[data-test-id="cost-estimator"]');
+    // with default main disk size: 120
+    let costEstimator = () => wrapper.find('[data-test-id="cost-estimator"]');
```

Comment:
a better pattern is to make this a function take a wrapper, instead of relying on the value it has when this function is defined.

Example: `const costEstimator = (wrapper) => wrapper.find('[data-test-id="cost-estimator"]');`